docs/puppeteer.md

```diff
@@ -1,5 +1,13 @@
 # Using Puppeteer with Lighthouse
 
+## Recipes
+
+### [Using Puppeteer for authenticated pages](./recipes/auth/README.md)
```

Comment:
would these be better as bullet points or something? IMO they look kind of weird as linked headings with no content, but the formatting is also not terribly important.

Reply:
I have no strong preference, I was just matching the style used for the custom gatherer one, and I see the value of bumping their visual prominence a bit over regular text-sized bullets.

Do they look weird enough to you to change? :) I'll accept whatever suggestion you put up

Reply:
> I see the value of bumping their visual prominence a bit over regular text-sized bullets.
> 
> Do they look weird enough to you to change? :) I'll accept whatever suggestion you put up

No suggestion, sorry, and I figured that was the reasoning. It is a little awkward looking but it's also functional and clear in what that function is, so that's why I said not terribly important :)